docs/api-reference/carto/cluster-tile-layer.md

```diff
@@ -105,11 +105,20 @@ The weight of each cell used for clustering.
 
 When using the `GeoJsonLayer` accessors to style the clusters, aggregated values will be passed to the styling accessor functions.
 
+### Valid aggregation types
+
 The type aggregation is infered based on the property name, for example `population_average` will be aggregated using a (mean) average operation across all the quadbin cells that are present in the cluster, while `age_min` will give the minimum value present in the cluster.
 
-### Valid aggregation types
+The following suffixes are supported:
 
-Supported aggregation types are: `any`, `average`, `count`, `min`, `max`, `sum`.
+| Suffix | Operation | Example |
+|--------|-----------|---------|
+| `_average` | Mean | `temperature_average` |
+| `_count` | Count | `point_count` |
+| `_min` | Minimum | `age_min` |
+| `_max` | Maximum | `age_max` |
+| `_sum` | Sum | `population_sum` |
+| `_any` | Any value | `category_any` |
```

Comment:
Optional – it does seem a bit inconsistent that we abbreviate "minimum" as "min" but not "average" as "avg". Would be a breaking change to rename now, I assume?

Reply:
It's what the API is doing, I think it is too much hassle to change now